src/main/kotlin/io/github/thewisenerd/linters/sidekt/rules/SQLQuerySniffer.kt

```diff
@@ -0,0 +1,68 @@
+package io.github.thewisenerd.linters.sidekt.rules
+
+import io.gitlab.arturbosch.detekt.api.*
+import org.jetbrains.kotlin.psi.KtClass
+import org.jetbrains.kotlin.psi.KtNamedFunction
+
+class SQLQuerySniffer(config: Config): Rule(config) {
+
+    private val sniffSQLAnnotation = listOf("SqlUpdate", "SqlCreate")
```

Comment:
Also add Select and Batch as well.

Reply:
Actually there was an SD in warehouse due to full table scan while SELECT.

Reply:
Done